src/Discord.Net.Rest/Entities/Invites/RestInvite.cs

```diff
@@ -62,6 +62,12 @@ internal void Update(Model model)
         public Task DeleteAsync(RequestOptions options = null)
             => InviteHelper.DeleteAsync(this, Discord, options);
 
+        /// <summary>
+        ///     Gets the URL of the invite.
+        /// </summary>
+        /// <returns>
+        ///     A string that is the URL of the invite.
```

Comment:
For `ToString`, we usually refer to `A string that resolves to X` if it points to another member of the class.